jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -0,0 +1,153 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+     options {
+        timeout(time: 10, unit: 'HOURS')
```

Comment:
Will this job take 10 hours?

Reply:
Not for the distribution builds - but the nightly builds, yes. 8-10 hours on average

Reply:
We need to then increase the time if the expected time is 10hrs, since this will fail the job after timeout

Reply:
We are still narrowing down on the exact numbers which we will know after the test runs. Currently we are looking at 8 hours. I can increase it later after getting in more datapoints.

Reply:
We'll definitely would want to revisit this. It would increase in case the job is run serially, but would decrease if run in parallel.